### DISCUSSION TITLE
Lago

### DISCUSSION THREAD

**Discussion by TudorAndrei** [coollabsio/coolify#2736] [Service Template Requests]
Can be selfhosted with docker 
https://getlago.com/docs/guide/self-hosted/docker

**Comment by ArtyomCZ** [comment]
+1

**Comment by wottpal** [comment]
+1

**Comment by jdenquin** [comment]
Hey guys, I'm from Lago, we'll take a look at it!

**Reply by shubhamchopade** [reply]
Did you get a chance to work on this?

**Comment by emregency** [comment]
Something weird happens when I try to deploy Lago with Coolify vs. local. During the first start, the migrate service prevents the completion of the deployment stalling the initialization.

migrate service log: `Database 'lago' already exists`
db service log: `ERROR:  database "lago" already exists | 
STATEMENT:  CREATE DATABASE "lago" ENCODING = 'utf8'`


@jdenquin - should I try to set LAGO_CREATE_ORG env variable to true? btw. I am also trying to get in touch with the sales team and book a demo, unsuccessfully.

**Comment by emregency** [comment]
I did try. Couldn’t succeed.On 11. Sep 2025, at 00:57, Shubham Chopade ***@***.***> wrote:﻿
Did you get a chance to work on this?
